http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -940,13 +948,12 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
           )
       }
 
-      dbAppOpt <- KubernetesServiceDbQueries.getActiveFullAppByName(msg.project, msg.appName).transaction
+      dbAppOpt <- KubernetesServiceDbQueries.getFullAppByName(msg.project, msg.appId).transaction
```

Comment:
With the new change, A lot of options indicate that the operation/step has already been completed so no action is necessary. Here, however, there is also the  possibility the app does not exist in the database. I changed this query so this option is purely representative of whether it exists in the database, even if it is deleted, and  future steps are idempotent.